KMB/make_KMB_info.py

```diff
@@ -476,13 +524,77 @@ def get_description(self):
         else:
             raise NotImplementedError
 
+    def make_commonscat_categories(self, cache):
+        """
+        Set categories based on depicted BBR/FMIS entries.
+
+        Sets a commonscat if one is found else a fallback category for the
+        object type.
+
+        Populates self.content_cats and modifies self.needs_place_cat.
+
+        :param cache: cache for category existence
+        :return: bool whether a comonscat was found
+        """
+        commonscat_map = self.kmb_info.mappings['commonscat']
+
+        found_commonscat = False
+        for fmis_id in self.fmis:
+            if fmis_id in commonscat_map['fmis']:
+                found_commonscat = True
+                self.content_cats.add(commonscat_map['fmis'][fmis_id]['cat'])
+            else:
+                self.make_default_fmis_category(cache)
+        for bbr_id in self.bbr:
+            if bbr_id in commonscat_map['bbr']:
+                found_commonscat = True
+                self.content_cats.add(commonscat_map['bbr'][bbr_id]['cat'])
+            else:
+                self.content_cats.add('Listed buildings in Sweden')
+
+        if found_commonscat:
+            self.needs_place_cat = False
+
+        return found_commonscat
+
+    def make_default_fmis_category(self, cache):
+        """
+        Set FMIS default categories based municipality.
+
+        Makes a guess for the category name. If not found it defaults to
+        County (Län) + Land (Landskap) categories.
+
+        Populates self.content_cats and modifies self.needs_place_cat.
+
+        :param cache: cache for category existence
+        """
+        test_cat = None
+        if self.kommunName:
+            test_cat = 'Archaeological monuments in {} Municipality'.format(
+                self.kommunName)
+            if not self.kmb_info.category_exists(test_cat, cache):
+                test_cat = 'Archaeological monuments in {}'.format(
+                    self.kommunName)
```

Comment:
I wonder if this can ever end up badly. I.e. are there cases when [stuff in $place] would _not_ be a good substitute for [stuff in $place Municipality]? It would require there existing a non-municipal place that just happens to share a name with a municipal one, but I can't think of an example off the top of my head. Quite possibly not an actual problem in practice?